examples/green-house-migration/Makefile

```
@@ -0,0 +1,266 @@
+.PHONY: help install dev test lint format clean run docker-build docker-run setup-dev db-migrate db-rollback docs-serve docs-build benchmark security-scan quality-report backup-data cleanup-old-data check-env generate-mock-data docker-clean docker-logs health-check dev-setup teamtailor-test teamtailor-migrate teamtailor-export
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)